iroha_permissions_validators/src/lib.rs

```diff
@@ -16,189 +17,277 @@ pub mod public_blockchain {
     /// A preconfigured set of permissions for simple use cases.
     pub fn default_permissions() -> PermissionsValidatorBox {
         PermissionsValidatorBuilder::new()
-            .with_recursive_validator(TransferOnlyOwnedAssets.into())
-            .with_recursive_validator(UnregisterOnlyAssetsCreatedByThisAccount.into())
-            .with_recursive_validator(MintOnlyAssetsCreatedByThisAccount.into())
-            .with_recursive_validator(BurnOnlyOwnedAssets.into())
-            .with_recursive_validator(BurnOnlyAssetsCreatedByThisAccount.into())
+            .with_recursive_validator(transfer::OnlyOwnedAssets)
+            .with_recursive_validator(unregister::OnlyAssetsCreatedByThisAccount)
+            .with_recursive_validator(mint::OnlyAssetsCreatedByThisAccount)
+            .with_recursive_validator(burn::OnlyOwnedAssets)
+            .with_recursive_validator(burn::OnlyAssetsCreatedByThisAccount)
+            .with_recursive_validator(keyvalue::RemoveOnlyCreatedByThisAccount)
+            .with_recursive_validator(keyvalue::SetOnlyCreatedByThisAccount)
```

Comment:
Please import according to our code style.

Reply:
It barely makes sense here, as I will just reimport it like this:
```
use transfer::OnlyOwnedAssets as TransferOnlyOwnedAssets;
use keyvalue::{
    RemoveOnlyCreatedByThisAccount as KeyValueRemoveOnlyCreatedByThisAccount,
    SetOnlyCreatedByThisAccount as KeyValueSetOnlyCreatedByThisAccount,
};
...
```

Reply:
Ok, I guess